src/main.ts

```diff
@@ -8,12 +10,25 @@ type Reaction = typeof REACTIONS[number];
 
 async function run() {
   try {
-    const message: string = core.getInput('message');
+    let message: string = core.getInput('message');
```

Comment:
```suggestion
    const message: string = core.getInput('message');
```

I'd prefer here to let the input variable let as it is. 
Prefer creating a new variable that would be something like : `body` (defaults to `= message`) and override this one in case of the presence of `fielPath`